apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/concurrent/WriterThreadPool.java

```diff
@@ -129,7 +130,16 @@ public long getSequenceNumberForObject(TMessage object) {
 
   @Nonnull
   public TClaim transformMessage(String apiVersion, TMessage message) {
-    return sink.transformMessage(apiVersion, message);
+    try {
+      return sink.transformMessage(apiVersion, message);
+    } catch (DataTransformer.TransformationException transformationException) {
+      try {
+        sink.writeError(message, transformationException);
+      } catch (IOException e) {
+        transformationException.addSuppressed(e);
```

Comment:
Seems like a great way to handle the failure to write the error with minimal .  The info will be in the logs but the rest of the shutdown process can happen normally.